Challenges - (30/30)

kobruhhhhh opened the pull request:
Challenge-10 completed....

shivaylamba committed:
parent 40e3697f70515ea3c713e69bcc8e52e9efd98e3f
author Shivay Lamba <shivaylamba@gmail.com> 1662010429 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663050543 +0530

author Shivay Lamba <shivaylamba@gmail.com> 1662010429 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663050541 +0530

author Shivay Lamba <shivaylamba@gmail.com> 1662010429 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663050538 +0530

author Shivay Lamba <shivaylamba@gmail.com> 1662010429 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663050527 +0530

author Shivay Lamba <shivaylamba@gmail.com> 1662010429 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663050487 +0530

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

This reverts commit 0c22dd76693b570e961d5b8972156e548425d9f2.

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

TeralaShivaKumar commented:
Hello kobruhhhhh
Congratulations on successfully completing the milestone 1 of opensource challenge
You will receive your goodies soon and we appreciate your consistency
#keep coding

kobruhhhhh committed:
added name in challengers-list.md

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

kobruhhhhh committed:
added gist-solutions.md

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

kobruhhhhh committed:
Challenge 19

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

kobruhhhhh committed:
parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941557 +0530

 parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941554 +0530

 parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941552 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941551 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941549 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941547 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941546 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941544 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941541 +0530

parent e02e774f3d1e78746c9258a3f651d2f89440cafa
author Lalit Kumar <77445506+kobruhhhhh@users.noreply.github.com> 1663856392 +0530
committer kobruhhhhh <kr.lalit.4528@gmail.com> 1663941538 +0530

Signed-off-by: kobruhhhhh <kr.lalit.4528@gmail.com>

kobruhhhhh commented:
# Challenge 22
**Screenshots of the site**

<img width="960" alt="image" src="https://user-images.githubusercontent.com/77445506/191910313-50e2e9b4-df1a-4092-b88d-96bec61fff69.png">

<img width="802" alt="image" src="https://user-images.githubusercontent.com/77445506/191910484-3ada343e-49d2-443d-a16c-103259f927a6.png">

kuraakhilesh8230 commented:
sry you haven't yet completed miletone-2 because there are too many commits pls correct it and you have till sunday time

kobruhhhhh committed:
Merge branch 'main' into kobruhhhhh-details

kuraakhilesh8230 commented:
congrats you have completed the milestone-2 successfully and you are good to go🎉
